src/app/core/data/object-updates/object-updates.service.ts

```diff
@@ -194,6 +204,25 @@ export class ObjectUpdatesService {
   saveChangeFieldUpdate(url: string, field: Identifiable) {
     this.saveFieldUpdate(url, field, FieldChangeType.UPDATE);
   }
+  isSelectedVirtualMetadata(url: string, relationship: string, item: string): Observable<boolean> {
```

Comment:
Same here, please add TypeDocs to this method

Reply:
I've updated the docs for the ObjectUpdates service